esmvaltool/utils/cmorizers/obs/utilities.py

```diff
@@ -151,6 +151,7 @@ def save_variable(cube, var, outdir, attrs, **kwargs):
     status = 'lazy' if cube.has_lazy_data() else 'realized'
     logger.info('Cube has %s data [lazy is preferred]', status)
     iris.save(cube, file_path, fill_value=1e20, **kwargs)
+    return file_name
```

Comment:
is this needed?

Reply:
same as above comment. It is a leftover from previous version that now I removed.